dns/nixos.org.js

```diff
@@ -0,0 +1,188 @@
+	CNAME("live", "caliban.nixos.org."),
+	CNAME("matrix", "caliban.nixos.org."),
+	CNAME("survey", "caliban.nixos.org."),
+	CNAME("vault", "caliban.nixos.org."),
+	DMARC_BUILDER({
+		label: "caliban",
+		policy: "none"
+	}),
+	SPF_BUILDER({
+		label: "caliban",
+		parts: [
+			"v=spf1",
+			"ip4:65.109.26.213",
+			"ip6:2a01:4f9:5a:186c::2",
+			"~all"
+		]
+	}),
+	TXT("mail._domainkey.caliban", "v=DKIM1; k=rsa; p=MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQDDCLtvNH4Ly+9COXf7InptMvoA7I5O347D7+j+saECt7RRe8yNz4TmhJTyJik+bg7e3+l7EJM0vE6k7xtpGBXACY6CCmg/8EgUi6YnDd126ttJHWpoqO96w4SWX93G+ZnoSC8O5rTPqdaTTkntYDTrw5u5n+7RA8GarZadgmaEzwIDAQAB"),
+
+	A("umbriel", "37.27.20.162"),
+	AAAA("umbriel", "2a01:4f9:c011:8fb5::1"),
+	// See `nixos.org.mail.key` in `non-critical-infra/modules/mailserver/default.nix`.
+	TXT("mail._domainkey", "v=DKIM1; k=rsa; p=MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQDcgNq4+Y23GxN8Mdza437tL5DuJJZU1y6VzTCwSi6cBNLyBDci2cmqXx/gm1sA3yv7+h+8/OyJpEgcbCIW/Ygs1XLuECqvXVX8MU6Djn4KY+d2sU1tlUdqvNM86puoneQtjEv9rDsjf3HGqaeOcjetFnQW7H+qcNcaEShxyKztzQIDAQAB"),
+
+	// ngi
+	CNAME("cache.ngi0", "d2tu257wv37zz1.cloudfront.net."),
+	CNAME("_293364b7f7ebb076ac287cd132f8b316.cache.ngi0", "_6a75cfb0c20f4eaac96b72afaffb489b.auiqqraehs.acm-validations.aws."),
+
+	A("makemake.ngi", "116.202.113.248"),
+	AAAA("makemake.ngi", "2a01:4f8:231:4187::"),
+	CNAME("buildbot.ngi", "makemake.ngi.nixos.org."),
+	CNAME("hydra.ngi0", "makemake.ngi.nixos.org."),
+	CNAME("summer", "makemake.ngi.nixos.org."),
+
+	A("tracker.security", "188.245.41.195"),
+	AAAA("tracker.security", "2a01:4f8:1c1b:b87b::1"),
+
+	// merge-bot
+	A("nixpkgs-merge-bot", "37.27.11.42"),
+	AAAA("nixpkgs-merge-bot", "2a01:4f9:c012:7615::1"),
+	A("nixpkgs-merge-bot-staging", "37.27.197.11"),
+	AAAA("nixpkgs-merge-bot-staging", "2a01:4f9:c010:dd30::1"),
+
+	// wiki
+	A("wiki", "65.21.240.250"),
+	AAAA("wiki", "2a01:4f9:c012:8178::"),
+	DMARC_BUILDER({
+		label: "wiki",
+		policy: "none"
+	}),
+	SPF_BUILDER({
+		label: "wiki",
+		parts: [
+			"v=spf1",
+			"ip4:65.21.240.250",
+			"ip6:2a01:4f9:c012:8178::",
+			"~all"
+		]
+	}),
+	TXT("mail._domainkey.wiki", "v=DKIM1; k=rsa; p=MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQDa+KjIljYr3q5MWWK7sEYzjR8OcA32zBh9BCPo6/HlY1q2ODTYsmE/FDZWpYMzM5z+ddnuGYdXia322XnZaNpZNoq1TbGYuQ5DsgAEK09CGoLuzONg3PSXTrkG7E2Sd6wstwHGJ5FHxSLKtNoWkknt9F5XAFZgXapO0w54p+BWvwIDAQAB"),
+
+	// cloudflare pages
+	CNAME("20th", "20th-nix.pages.dev."),
+
+	// github org/domain binding
+	TXT("_github-challenge-nixos", "9e10a04a4b"),
+
+	// github pages
+	CNAME("mobile", "nixos.github.io."),
+	CNAME("ngi", "ngi-nix.github.io."),
+	CNAME("reproducible", "nixos.github.io."),
+
+	TXT("_github-pages-challenge-ngi-nix.ngi", "4e8bffbb7ced2aec7be1f8cf3561d6"),
+	TXT("_github-pages-challenge-nixos", "f3a423ba6916e972cfb1e74f82f601"),
+
+	// netlify pages
+	CNAME("planet", "nixos-planet.netlify.app."),
+	CNAME("status", "nixos-status.netlify.app."),
+	CNAME("weekly", "nixos-weekly.netlify.com."),
+
+END);
```

Comment:
This is [no longer required](https://docs.dnscontrol.org/advanced-features/code-tricks#trailing-commas):

```suggestion
);
```

Reply:
Ah wild, was added by the `get-zone` command on 4.14.3.

Reply:
Yeah, I noticed that it still does that; going to open a PR to fix it.

Reply:
nvm, already fixed (and I was looking at an old version): https://github.com/StackExchange/dnscontrol/releases/tag/v4.15.2

Bump your local dnscontrol? ;)

Reply:
It's whatever the shell has, so NixOS 24.11.